fuzz/fuzz_targets/compat.rs

```diff
@@ -46,10 +46,22 @@ fuzz_target!(|data: &[u8]| {
     let bincode_v1: Result<AllTypes, _> = configv1.deserialize(data);
```

Comment:
switch this to `deserialize_from`. Right now this doesn't do _any_ limit checking and the fuzzer might cause OOM